src/Microsoft.AspNet.OData.Shared/Builder/IODataInstanceAnnotationContainer.cs

```diff
@@ -13,7 +13,7 @@ namespace Microsoft.AspNet.OData.Builder
 {
     /// <summary>
     /// Interface to used as a Container for holding Instance Annotations, An default implementation is provided
-    /// Custoer can implement the interface and can have their own implementation.
+    /// Customer can implement the interface and can have their own implementation.
```

Comment:
```suggestion
    /// Customers can have their own implementation.
```
=> "Customer can implement the interface" and "can have their own implementation" mean the same thing...